README.md

```diff
@@ -32,9 +34,60 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Deployment
+## Quick Installation via Helm
 
-To quickly deploy the latest development version of StackRox to your kubernetes
+StackRox offers quick installation via Helm Charts, follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) for system specific instructions and requirements. 
+First, add the [stackrox/helm-charts/opensource](https://github.com/stackrox/helm-charts/tree/main/opensource) repository to Helm.
+```sh
+helm repo add stackrox-oss https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+```
+To see all available Helm charts in the repo run (you may add the option `--devel` to show non-release builds as well)
+```sh
+helm search repo stackrox-oss
+```
+From here you can install stackrox-central-services to get Central and Scanner components deployed on your cluster. Note that you need only one deployed instance of stackrox-central-services even if you plan to secure multiple clusters.
+```sh
+helm install -n stackrox-oss --create-namespace stackrox-central-services stackrox-oss/stackrox-central-services --devel
+```
+To create a secured cluster, you first need to generate an init bundle containing initialization secrets. The init bundle will be saved in `stackrox-init-bundle.yaml`. Use it to provision secured clusters as shown below.
+```sh
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+    --password "$(kubectl -n stackrox get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}')" \
+    central init-bundles generate stackrox-init-bundle --output - > stackrox-init-bundle.yaml
+```
+Then install stackrox-secured-cluster-services in the same cluster using this command with the init bundle you just generated:
+```sh
+helm install -n stackrox-oss --create-namespace stackrox-secured-cluster-services stackrox-oss/secured-cluster-services \
+    -f stackrox-init-bundle.yaml \ 
+    --set clusterName=<name_of_the_secured_cluster> 
```

Comment:
1. Couple fixes
```suggestion
helm install -n stackrox stackrox-secured-cluster-services stackrox-oss/stackrox-secured-cluster-services \
    -f stackrox-init-bundle.yaml \
    --set clusterName=<name_of_the_secured_cluster> 
```

2. Again, for development/small clusters it make sense to update resources to be more conservative.
See development values here https://github.com/stackrox/stackrox/blob/3149711ad41787b732c6afb674e5384848af6c9b/deploy/common/k8sbased.sh#L533

However, unlike updating central services after they were deployed, upgrading secured-cluster-services is finicky. Better to append request values upon creation. Here's what I came up with:

```
        --set sensor.resources.requests.memory=500Mi \
        --set sensor.resources.requests.cpu=500m \
        --set sensor.resources.limits.memory=500Mi \
        --set sensor.resources.limits.cpu=500m
```

You can say something like:

> Append the following arguments to the above command in order to limit sensor resources usage if you're deploying on development or small clusters. Do not use these values on production clusters.